front/cypress/.eslintrc.js

```diff
@@ -79,6 +79,34 @@ module.exports = {
     'no-multiple-empty-lines': 'off',
     'no-new-wrappers': 'error',
     'no-param-reassign': 'error',
+    'no-restricted-imports': [
+      'error',
+      {
+        paths: [
+          {
+            name: 'react-router-dom',
+            importNames: ['Link', 'useNavigate', 'NavLink'],
+            message:
+              "Please import the Link or useNavigate from utils/cl-router instead of directly from 'react-router-dom'",
+          },
+          {
+            name: 'react-intl',
+            importNames: ['FormattedMessage', 'injectIntl'],
+            message:
```

Comment:
Let's make these messages consistent somehow - don't really mind any of the formats here but let's just pick one ;)